src/css/components/section.scss

```diff
@@ -64,6 +75,60 @@ $chevron-size: 4.865rem;
   }
 }
 
+.section-dark {
+  background-color: $color-textblack;
+
+  &:before {
+    background-image: url('#{$static-img-path}/chevron-top.svg');
+  }
+}
+
+.section-blue {
+  background-color: $color-blue;
+
+  a,
+  a:active,
+  a:hover,
+  a:visited {
+    color: $color-white;
+    text-decoration: underline;
+  }
+
+  &:before {
+    background-image: url('#{$static-img-path}/chevron-top-blue.svg');
+  }
+}
+
+.section-cream {
+  background-color: $color-cream;
+  color: $color-textblack;
+
+  a,
+  a:active,
+  a:hover,
+  a:visited {
+    color: $color-red;
+    text-decoration: underline;
+  }
+
+  a:active,
```

Comment:
This is an overwrite to prevent writing an separate `a, a:visited` rule for color, but I can write it that way if desired.